internal/website/themes/hyde/static/css/hyde.css

```diff
@@ -70,6 +70,11 @@ html {
   }
 }
 
+figure.aligncenter img {
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
As per [Google CSS style](https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties), I recommend rewriting this as `margin: 1em auto`.